src/Google/Ads/GoogleAds/Util/FieldMasks.php

```diff
@@ -155,9 +155,16 @@ private static function buildPaths(
                     $paths[] = $fieldName;
                 }
             } else {
+                $hasser = self::getHasser($fieldDescriptor->getName());
```

Comment:
This assumes that all fields use presence. I am not sure that we want to make this assumption?

Reply:
With the current protoc, it generates hasser for all fields anyway. Let's discuss this offline.

Reply:
Sure.

Quick clarification: this assumption could lead to issues if we do not regen all protos including older versions of the API because FieldMasks is not API version-specific.

Reply:
Thanks for reminding. I actually planned to add a condition to check if the method really exists too. Done now.